docs/rfc/0004_component_groups_scale_view.md

```diff
@@ -0,0 +1,300 @@
+|                    |                                                         |
+| -------            |---------------------------------------------------------|
+| Feature            | Expose the scale subresource with `RisingWaveScaleView` |
+| Status             | In-progress                                             |
+| Date               | 2022-09-05                                              |
+| Authors            | arkbriar                                                |
+| RFC PR #           |                                                         |
+| Implementation PR #|                                                         |
+|                    |                                                         |
+
+# **Summary**
+
+The RFC aims to provide a new CR to support the scale subresource over groups of some user-specified component of
+the RisingWave resource. With that, the users can benefit from the scale subresource and apply any tools that have such
+requirement. For example, the horizontal auto-scaling service (HPA/[KEDA](https://github.com/kedacore/keda)) will work
+after this is implemented.
```

Comment:
Why HPA and KEDA does not work for now?

Reply:
yes, they all require that the CR exposes the `scale` subresource.